marketplace/SafeBuyFractionalStrategy.tsx

```diff
@@ -132,12 +125,11 @@ export class SafeBuyFractionalStrategy extends BuyFractionalStrategy {
       throw new Error("Approval error");
     }
 
-    // TODO: this whole step should probably not be here
     try {
       await setStep("Transfer manager");
       const isTransferManagerApproved =
         await this.exchangeClient.isTransferManagerApprovedSafe(this.address);
-      // FIXME: this shouldn't be here, unless we're missing something
+
```

Comment:
Did this get resolved?

Reply:
I only added this FIXME when on a call with @bitbeckers because we didn't understand it. Since then you two have been working on this and releasing it so my understanding was, that this is actually the way it's supposed to be. Am I missing something?

Reply:
I don't think you're missing something. This looks correct to me.